catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/RoleRepository.java

```diff
@@ -180,24 +186,16 @@ public void storeRelationships(Role role) {
   }
 
   public ResultList<Role> getDefaultRolesResultList(UriInfo uriInfo, Fields fields)
-      throws GeneralSecurityException, UnsupportedEncodingException {
+      throws GeneralSecurityException, IOException {
     List<Role> roles = getDefaultRoles(uriInfo, fields);
     return new ResultList<>(roles, null, null, roles.size());
   }
 
-  private List<Role> getDefaultRoles(UriInfo uriInfo, Fields fields) {
-    List<Role> roles =
-        daoCollection.roleDAO().getDefaultRoles().stream()
-            .map(
-                json -> {
-                  try {
-                    return withHref(uriInfo, setFields(JsonUtils.readValue(json, Role.class), fields));
-                  } catch (IOException e) {
-                    LOG.warn("Could not parse Role from json {}", json);
```

Comment:
Wasn't happy about hiding this exception.
Replacing the lambda here with simple for loop.